docs/pages/reference/meshync-crd-reference.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: MeshSync CRD Reference
+abstract: "Meshync CRD Documentation and Reference"
+permalink: reference/meshync-crd
+redirect_from: reference/
+type: Reference
+language: en
+---
+
+# MeshSync CRD
+The MeshSync CRD is used as a configuration tool the the Meshsync service. The CRD is used to control the replica count for Meshsync instances, broker configuration and the resources that Meshsync watches and listens to in a cluster. 
```

Comment:
```suggestion
The MeshSync CRD is used as a configuration tool for the MeshSync service. The CRD is used to control the replica count for MeshSync instances, broker configuration and the resources that MeshSync watches and listens to in a cluster. 
```

Reply:
MeshSync service? I think this will create confusion. Can you describe more explicitly.

Reply:
Instead, just say “MeshSync”, yeah?  Also, a link to the existing Alesha sync page can help explain MeshSync’s purpose and overall behavior.